core/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -168,6 +168,74 @@ class DeltaTableSuite extends QueryTest
     }
   }
 
+  test("isDeltaTableByPath - with _delta_log dir") {
+    withTempDir { dir =>
+      testData.write.format("delta").save(dir.getAbsolutePath)
+      assert(DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with empty _delta_log dir") {
+    withTempDir { dir =>
+      new File(dir, "_delta_log").mkdirs()
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with no _delta_log dir") {
+    withTempDir { dir =>
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with non-existent dir") {
+    withTempDir { dir =>
+      JavaUtils.deleteRecursively(dir)
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with non-Delta table path") {
+    withTempDir { dir =>
+      testData.write.format("parquet").mode("overwrite").save(dir.getAbsolutePath)
+      assert(!DeltaTable.isDeltaTable(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with table name in current path") {
+    val currDir = System.getProperty("user.dir")
+    try {
+      testData.write.format("delta").save(currDir + "/deltaTableName")
+      assert(DeltaTable.isDeltaTableByPath("deltaTableName"))
+    } finally {
+      JavaUtils.deleteRecursively(new File(currDir, "deltaTableName"))
+    }
+  }
+
+  test("isDeltaTableByName - with table name") {
```

Comment:
can we add a false case for isDeltaTableByName? maybe a (a) parquet table (b) non-existent table or view?